package.json

```
@@ -9,10 +9,13 @@
   "isAwsLambda": true,
   "riffraffFile": "./riff-raff.yaml",
   "devDependencies": {
+    "@aws-sdk/client-dynamodb": "^3.489.0",
+    "@aws-sdk/client-lambda": "^3.489.0",
+    "@aws-sdk/client-s3": "^3.489.0",
+    "@aws-sdk/client-sts": "^3.489.0",
```

Our recommendation is to use pinned versions
```suggestion
    "@aws-sdk/client-dynamodb": "3.489.0",
    "@aws-sdk/client-lambda": "3.489.0",
    "@aws-sdk/client-s3": "3.489.0",
    "@aws-sdk/client-sts": "3.489.0",
```

---

It is, but that's not something that this repo has adopted by the looks, so the other dependencies aren't pinned like this. We could update the whole file to pinned deps in a follow-up so we don't end up with a mix?